src/navigation/StackNavigator/index.tsx

```diff
@@ -4,22 +4,45 @@ import {
 } from "react-navigation-stack";
 import CollectCustomerDetailsScreen from "./CollectCustomerDetailsScreen";
 import CustomerQuotaScreen from "./CustomerQuotaScreen";
+import MerchantPayoutScreen from "./MerchantPayoutScreen";
 
 const StackNavigator = createStackNavigator(
```

Comment:
Hmm, given that this flow is entirely separate from the existing flow, it shouldn't be in the same stack. So there should be 3 children of the main switch navigator.

Given that we're still working on the backend feature flagging, let's create a frontend feature flag for this entire feature. So on validating OTP, it navigates to the MerchantPayoutScreen instead https://github.com/rationally-app/mobile-application/blob/f24f219c497e5644d7bd8d47f3bf096d3b0278d7/src/components/Login/LoginOTPCard.tsx#L73

In that case, we don't need to add a new section in `CollectCustomerDetailsScreen`

Reply:
Hmm ist not possible to add 3 items to a switch navigator, I tried and it throws a AppContainter null error!